apps/meteor/tests/end-to-end/api/05-chat.js

```diff
@@ -785,44 +791,40 @@ describe('[Chat]', function () {
 			});
 
 			it('should have an iframe oembed with style max-width', (done) => {
-				setTimeout(() => {
-					request
-						.get(api('chat.getMessage'))
-						.set(credentials)
-						.query({
-							msgId: ytEmbedMsgId,
-						})
-						.expect('Content-Type', 'application/json')
-						.expect(200)
-						.expect((res) => {
-							expect(res.body).to.have.property('message').to.have.property('urls').to.be.an('array').that.is.not.empty;
+				request
+					.get(api('chat.getMessage'))
+					.set(credentials)
+					.query({
+						msgId: ytEmbedMsgId,
+					})
+					.expect('Content-Type', 'application/json')
+					.expect(200)
+					.expect((res) => {
+						expect(res.body).to.have.property('message').to.have.property('urls').to.be.an('array').that.is.not.empty;
 
-							expect(res.body.message.urls[0])
-								.to.have.property('meta')
-								.to.have.property('oembedHtml')
-								.to.have.string('<iframe style="max-width: 100%;width:400px;height:225px"');
-						})
-						.end(done);
-				}, 1000);
+						expect(res.body.message.urls[0])
+							.to.have.property('meta')
+							.to.have.property('oembedHtml')
+							.to.have.string('<iframe style="max-width: 100%;width:400px;height:225px"');
+					})
+					.end(done);
 			});
 
 			it('should embed an image preview if message has an image url', (done) => {
-				setTimeout(() => {
-					request
-						.get(api('chat.getMessage'))
-						.set(credentials)
-						.query({
-							msgId: imgUrlMsgId,
-						})
-						.expect('Content-Type', 'application/json')
-						.expect(200)
-						.expect((res) => {
-							expect(res.body).to.have.property('message').to.have.property('urls').to.be.an('array').that.is.not.empty;
+				request
```

Comment:
same concern as above.. 

I wonder though if we should change both these end-to-end tests for some unit tests.. what we need is to check if the oembed code is doing it's thing.. relying on external services (like on the youtube link above) is prone to fail.. wdyt?

Reply:
Agreed, I'll remove (skip) these tests in this PR and I will create that in another PR, wdyt?